### DISCUSSION TITLE
State of funding for iOS port

### DISCUSSION THREAD

**Discussion by burrscurr** [streetcomplete/StreetComplete#5884] [General]
I read in the [SOTM slides](https://docs.google.com/presentation/d/11_k8Tf33xZ0fI2gc3yaMJAkXeQBbLzw0Be5m28P_oYI) that the (primary) funding for the iOS port has ended in August. If I understood it correctly, this funding has allowed @westnordost to work (full-time?) on SC development. Are there other/additional sources of funding for the next months? What does this mean for the pace of development, and is there something one can do to support (that is not already mentioned in #5421)?

**Comment by westnordost** [comment]
Yes, I was able to work (quasi) full-time on it for six months. I have no updates beyond what I wrote in #5421 right now.
